app/views/users/_admins.html.erb

```diff
@@ -9,10 +9,12 @@
   </thead>
   <tbody>
     <% admins.each do |admin| %>
-      <tr>
+      <%= content_tag_for(:tr, admin) do %>
         <td><%= admin.name %></td>
-        <td class="table-actions"><a href="" class="table-link table-delete">Delete</a></td>
-      </tr>
+        <td class="table-actions">
+          <%= render("users/delete", user: admin) %>
```

Comment:
I've decided to be deliberately explicit about partial names in this project -- it's frustrating when refactoring/renaming a parent partial breaks children partials.